tests/spread/core22/remove-hook/task.yaml

```diff
@@ -0,0 +1,61 @@
+summary: Verify snapcraft remove hook deletes base images and instances
+
+prepare: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  set_base "snap/snapcraft.yaml"
+
+restore: |
+  #shellcheck source=tests/spread/tools/snapcraft-yaml.sh
+  . "$TOOLS_DIR/snapcraft-yaml.sh"
+  restore_yaml "snap/snapcraft.yaml"
+
+  # reinstall snapcraft
+
+  # If $SNAPCRAFT_CHANNEL is defined, install snapcraft from that channel.
```

Comment:
Copied verbatim from `spread.yaml`'s project `prepare`.   I think I should move this to a function and call it to avoid duplication.